client/components/forms/form-text-input-with-action/style.scss

```diff
@@ -0,0 +1,44 @@
+.form-text-input-with-action {
+	@extend %form-field;
+	display: flex;
+	padding: 0;
+
+	&.is-focused {
+		border-color: $blue-wordpress;
+		outline: none;
+		box-shadow: 0 0 0 2px $blue-light;
+	}
+
+	&.is-disabled {
+		background: $gray-light;
+		border-color: lighten( $gray, 30% );
+		color: lighten( $gray, 10% );
+		opacity: 1;
+		-webkit-text-fill-color: lighten( $gray, 10% );
+
+		&:hover {
+			cursor: default;
+		}
+
+		&::placeholder {
+			color: lighten( $gray, 10% );
+		}
+	}
+}
+
+.form-text-input-with-action__button.is-compact {
+	flex: 1 0 auto;
```

Comment:
![screen shot 2016-10-14 at 14 29 12](https://cloud.githubusercontent.com/assets/908665/19389139/a21366fe-921a-11e6-86da-a78414c8ae5f.png)

We don't want the button to glow, I think the first value should be`0` as `flex: 0 0 auto;`